### DISCUSSION TITLE
Tests for Databrowser

### DISCUSSION THREAD

**Discussion by gappc** [noi-techpark/it.bz.opendatahub.databrowser#141] [General]
- [ ] how to test (manual vs. automatic)
- [ ] define the use case to be tested
- [ ] define the OS, devices and browser to be tested
- [ ] what system / utils to use
- [ ] where to execute (local, remote, as-a-service)
- [ ] ....

**Comment by mrabans** [comment]
See here for a basic automated testing 
https://automate.browserstack.com/dashboard/v2/quick-start/get-started
